react/src/Calendar/CalendarBase/CalendarContext.tsx

```diff
@@ -240,7 +257,7 @@ const useProvideCalendar = ({
         setCurrYear(newDate.getFullYear())
       }
     },
-    [currMonth, currYear, monthsToDisplay],
+    [currMonth, currYear, monthsToDisplay, setCurrMonth, setCurrYear],
   )
 
   /**
```

Comment:
We may also want to update isDateFocusable to be false when `isLoading=true`

Reply:
went with an alternative approach of wrapping the whole button with Skeleton, so no button props / styling need to be conditional on `isLoading`